geom/dPoly.cpp

```diff
@@ -1412,7 +1417,26 @@ void dPoly::writePoly(std::string filename, std::string defaultColor) {
 
   return;
 }
+void dPoly::clearExtraData(){
```

Comment:
One can add a comment here saying that this must be used each time the structure of the polygon changes